### PR TITLE
BC-7415 - Make sidebar usable on tablets

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Short Description
- sidebar not scrollable on tablet with first touch/scroll (vue)
- scroll on main content not disabled, when sidebar is open (vue and legacy)
- scroll on task item more menu (in task overview) not possible when "edit" option is touched (vue)

## Links to Ticket and related Pull-Requests
BC-7415
https://github.com/hpi-schul-cloud/schulcloud-client/pull/3457

## Changes
(see Description)

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/ui/layout/sidebar/SidebarCategoryItem.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
 			</v-list-item>
 		</template>
 		<template v-for="child in item.children" :key="child.title">
-			<SidebarItem :item="child" />
+			<SidebarItem :item="child" :draggable="false" />
```
Comment (odalys-dataport) [inline src/modules/ui/layout/sidebar/SidebarCategoryItem.vue:19]: Where does this draggable prop come from?

File: src/layouts/newLoggedIn.layout.vue [hunk 2]
```diff
@@ -30,15 +33,25 @@ import LoadingStateDialog from "@/components/molecules/LoadingStateDialog.vue";
...
+});
+
 const sidebarExpanded = ref(isDesktop.value);
 
 const onToggleSidebar = () => {
 	sidebarExpanded.value = !sidebarExpanded.value;
 };
 </script>
+<style lang="scss" scoped>
+.fixed-position {
```
Comment (odalys-dataport) [inline src/layouts/newLoggedIn.layout.vue:53]: I'd rather use `!isDesktop` and vuetify's utility classes here `position-fixed w-100`, but it's fine :D

File: src/modules/ui/layout/sidebar/SidebarCategoryItem.vue (hunk 1 above)
Comment (MartinSchuhmacher) [reply src/modules/ui/layout/sidebar/SidebarCategoryItem.vue:19]: vuedraggable uses that option to not make a component draggable

File: src/layouts/newLoggedIn.layout.vue (hunk 2 above)
Comment (MartinSchuhmacher) [reply src/layouts/newLoggedIn.layout.vue:53]: uhh thats a good point, thx, will adjust :)

File: src/modules/ui/layout/sidebar/SidebarCategoryItem.vue (hunk 1 above)
Comment (odalys-dataport) [reply src/modules/ui/layout/sidebar/SidebarCategoryItem.vue:19]: I'm very curious about this, as the Sidebar doesn't use vuedraggable

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**